solr/solrj/src/java/org/apache/solr/common/util/BytesOutputStream.java

```diff
@@ -22,6 +22,7 @@
 import java.io.UnsupportedEncodingException;
 import java.util.Arrays;
 
+@Deprecated
```

Comment:
unused

Reply:
could we add the since tag to help us remember when to remove this deprecated method?   There are so many deprecated methods that just kind of hang around because we don't have good mechanisms for deciding/remembering to remove them!

Reply:
I deleted it; we don't need to wait for internal utilities